kong.conf.default

```diff
@@ -337,7 +390,7 @@
                                  # significant memory allocations to process
                                  # many concurrent large request bodies.
 
-# Note: see http://nginx.org/en/docs/http/ngx_http_core_module.html#client_body_buffer_size
+# Note: see [the Nginx docs](http://nginx.org/en/docs/http/ngx_http_core_module.html#client_body_buffer_size)
```

Comment:
Better not to include such advanced markdown formatting in this file (other links are plain, e.g. the docs on `lua_package_path`.

Reply:
Ok, removed